advantages

VitaliiPankin opened the pull request:
сделал секцию

VitaliiPankin committed:
advantages

сделал секцию

Yaroslav-Zozulya commented:
убери стили контейнера 
 убрать font-style: normal
 убрать font-weight: 400
 убрать line-height: calc. Используем множитель.

VitaliiPankin committed:
правки

Yaroslav-Zozulya commented:
- убрать изменения из спрайта.
- Иконки преимуществ в макете отрисованы в png формате. В вёрстке их нужно реализовать псевдоэлементами. У вас реализовано через спрайт. Визуально всё отлично но по ТЗ не проходит.
- line-height для advantages__content 1.67 (не округлили просто)
- line-height для  advantages__label 1.39

VitaliiPankin committed:
переделал

ну смотри спрайт я убрал. но через бефор я не могу подключить svg из спрайта по этому они в отдельной папке
